nestkernel/rng_manager.cpp

```diff
@@ -280,8 +278,6 @@ nest::RNGManager::create_rngs_()
 void
 nest::RNGManager::create_grng_()
 {
-  // create new grng
-  LOG( M_INFO, "Network::create_grng_", "Creating new default global RNG" );
```

Comment:
The output in this line and the one mentioned above make `regressiontests/issue-779-1016.py` fail on my machine (the test captures output and compares it to some reference).

As the error only started since the changes in this PR I thought that problem might also affect other users and removed them (they were not that helpful, were they?).

Reply:
@jougs This is a bit strange. The test ([see here](https://github.com/nest/nest-simulator/blob/2c9d73f2306d391902719c8bc88c851811869b7f/testsuite/regressiontests/issue-779-1016.py#L49)) should print the argv from statusdict after the kernel has completely started, so the CreateRNG messages should come first. Why do they come later on your machine? Interestingly, they cannot even be suppressed using `--verbosity=FATAL`, since they apparently are executed before verbosity has been handled.

But maybe these lines here should go, after all, `Create()` doesn't print a message "new neurons created" either.

Reply:
Argh, you are right!

I was pointing you to the wrong test and even got the context all wrong. I meant [test_connect_all_patterns.py](https://github.com/nest/nest-simulator/pull/1071/files#diff-5e7934b0517f89b433502dd81359c5f6R27) and [test_mpitests.py](https://github.com/nest/nest-simulator/pull/1071/files#diff-e8570780ae94f31e498d275238e8083eR27). I the linked lines I need to get the output of NEST used in pure command-mode. And as you correctly observed, these messages cannot be silenced at all, because they are printed from some constructor long before the verbosity setting is actually taken into account.

See also #415 and #700 (which I've now added to the description of this one).